anvio/db.py

```diff
@@ -496,6 +496,9 @@ def get_table_as_dict(self, table_name, table_structure=None, string_the_key=Fal
         # new ones to avoid key information to not be overwritten due to the lack of unique entry ids which become keys for
         # the data dictionary). in other cases there are no ways to fix it, such as for HMM tables.. The ACTUAL SOLUTION to\
         # this is to remove `entry_id` columns from every table in anvi'o, and using SQLite indexes as entry ids.
+        #
+        # NOTE from the future
+        # Every SQLite table has an implicit column called ROWID. Does this solve our problem?
```

Comment:
yes, but unfortunately I learned about it a bit too late :/

Reply:
it would have made things MUCH more simpler if I had used this instead of `entry_id` monstrosity..